### ISSUE TITLE
Only a single tab is moved

### ISSUE THREAD

**Issue by Suika** [stefanXO/Tab-Manager-Plus#44]
For some reason multiple tabs cannot be moved to a new window.
Selecting a few tabs and clicking "move to a new window" results only in a single tab being moved.
Be it active, inactive or mixed tabs. No other addon is active, except Tab-Manager-Plus.

FF 72.0b6 (64-Bit)
Tab-Manager-Plus 5.1.3

Debug logs says:
```
sendRemoveListener on closed conduit {45f2dc53-96cd-4c41-91f6-f4a73a8fb2b0}.549755881416 ConduitsChild.jsm:108
    _send resource://gre/modules/ConduitsChild.jsm:108
    _send self-hosted:977
    removeListener resource://gre/modules/ExtensionChild.jsm:1095
    removeListener resource://gre/modules/ExtensionChild.jsm:1320
    onChanged chrome://extensions/content/child/ext-storage.js:333
    removeListener resource://gre/modules/ExtensionCommon.jsm:2530
    revoke resource://gre/modules/ExtensionCommon.jsm:2552
    close resource://gre/modules/ExtensionCommon.jsm:2557
    unload resource://gre/modules/ExtensionCommon.jsm:896
    unload resource://gre/modules/ExtensionPageChild.jsm:261
    unload resource://gre/modules/ExtensionPageChild.jsm:310
    destroyExtensionContext resource://gre/modules/ExtensionPageChild.jsm:494
    observe resource://gre/modules/ExtensionPageChild.jsm:407
```

```
TypeError: can't access dead objectbackground.js:12:1
```

**Comment by Suika** [comment]
Tested the drag and drop of selected tabs. It works without a problem.
Yet the clicking of "move to a new window" does not work.

**Comment by pgoodwin** [comment]
I have the exact same problem.

**Comment by yozlet** [comment]
@pgoodwin Which browser, version and OS are you using?

I'm on Chrome 79 (Mac) and it works fine for me.

**Comment by mastef** [comment]
Looks like maybe a FireFox issue. It worked only in one of 6 tests I ran on FF. Seems to be working in Chrome correctly

**Comment by mastef** [comment]
Could you check if this version fixes it for you? https://github.com/stefanXO/Tab-Manager-Plus/issues/62#issuecomment-614399625

**Comment by mastef** [comment]
Should be fixed with 5.1.4 which was just submitted to the Firefox Add-Ons store. Let me know if you encounter any issues with it
